libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1216,3 +1216,18 @@ AP_GPS_UBLOX::broadcast_configuration_failure_reason(void) const {
         }
     }
 }
+
+void AP_GPS_UBLOX::Write_DataFlash_Log_Startup_messages() const
+{
+    AP_GPS_Backend::Write_DataFlash_Log_Startup_messages();
+    if (is_configured()) {
```

Comment:
Actually I don't think this part should be logged to DF at all. This is solely meant for arming check purposes? We don't log once we configure a message, so I could see this causing some wasted time for log analysis purposes.

What we should do is add the _unconfigured_messages field to one of the UBX messages that gets logged instead. That way we know continuously for a flight, as well get feedback if anything is ever fixed while in flight as well.